PostgreSQL/node.prepare.js

```diff
@@ -0,0 +1,19 @@
+let logger = global.logger;
+
+module.exports = async (connectorData, inputData) => {
+	logger.trace("PostgreSQL Prepare Node: Invoked!");
+	try {
+		logger.trace(`PostgreSQL Prepare Node: input - ${JSON.stringify(inputData)}`);
```

Comment:
Add check if query exists.

Reply:
FIxed. Instead of throwing an error, we will simply run an empty query which does not perform any action.

Reply:
Okay on second thoughts, to use consistent checks everywhere, throwing an error in case query is missing.